javascripts/discourse/components/dc-sidebar-subcategories.js.es6

```diff
@@ -0,0 +1,34 @@
+import Component from "@ember/component";
+import { computed } from "@ember/object";
+
+export default Component.extend({
+  tagName: "ul",
+  classNames: ["ml-4 list-unstyled list-nested"],
+  classNameBindings: ["isExpanded:expanded"],
+
+  isExpanded: false,
+
+  category: null,
+  currentCategoryId: null,
+
+  didReceiveAttrs() {
+    this._super(...arguments);
+    this._computeAttrs();
+  },
+
+  _computeAttrs() {
+    // Avoid to do any comparation unless there is value
+    if (!this.currentCategoryId) return;
```

Comment:
why we cannot check this anyway below and return false?

Reply:
Thanks, this actually inject the bug I was having when going towards "all categories" and not collide the already expanded parent category